chips/stm32f4xx/src/clocks/clocks.rs

```diff
@@ -621,6 +621,7 @@ impl<'a, ChipSpecs: ChipSpecsTrait> Stm32f4Clocks for Clocks<'a, ChipSpecs> {
 ///
 /// If there are any errors, open an issue ticket at <https://github.com/tock/tock>. Please provide the
 /// output of the test execution.
+#[allow(clippy::wildcard_imports)]
 pub mod tests {
     use super::*;
```

Comment:
I don't think we should add this. I get that it's only for tests, but the same issues with `import *` are still there in that it is not clear what is needed for testing. This shouldn't have been merged in the first place.